NEWS.md

```diff
@@ -14,7 +14,8 @@ Language changes
     returns a `BigFloat` with precision equal to `precision(BigFloat)` ([#29127]).
   * Parser inputs ending with a comma are now consistently treated as incomplete.
     Previously they were sometimes parsed as tuples, depending on whitespace ([#28506]).
-  * `Regex` now behave like a scalar when used in broadcasting ([#29913]).
+  * `Regex` now behaves like a scalar when used in broadcasting ([#29913]).
+  * `Char` now behaves like a 0-dimensional array ([#29819]).
```

Comment:
Isn't this a bit of a strong claim?
```
julia> A = fill(0);

julia> A[] = 1
1

julia> c = 'a';

julia> c[] = 'b'
ERROR: MethodError: no method matching setindex!(::Char, ::Char)
```

Reply:
I have added information that it is read-only.